HW13/proxy/proxy_txt_writer.py

```diff
@@ -0,0 +1,18 @@
+from HW13.proxy.txt_writer import TxtWriter
+
+
+class TxtProxyWriter:
+    def __init__(self, file_path):
+        self.__result = None
+        self.__txt_writer = TxtWriter(file_path)
+
+    def write_file(self, new_data):
+        self.__result = self.__txt_writer.write(new_data)
+        return self.__result
```

Comment:
and where is proxy?
In some cases file should be written in some not. Please think about in in scope of file content